plume-models/src/blogs.rs

```diff
@@ -409,7 +298,106 @@ impl IntoId for Blog {
     }
 }
 
-impl WithInbox for Blog {
+impl FromId<PlumeRocket> for Blog {
+    type Error = Error;
+    type Object = CustomGroup;
+
+    fn from_db(c: &PlumeRocket, id: &str) -> Result<Self> {
+        Self::find_by_ap_url(&c.conn, id)
+    }
+
+    fn from_activity(c: &PlumeRocket, acct: CustomGroup) -> Result<Self> {
+        let url = Url::parse(&acct.object.object_props.id_string()?)?;
+        let inst = url.host_str()?;
+        let instance = Instance::find_by_domain(&c.conn, inst).or_else(|_| {
```

Comment:
out of scope for this pr, but we should consider a `find_or_create` of some kind instead of `find_by_domain.or_else(insert)`, so to not tie to much code from one part of the model to another